iphone/Classes/ImageLoader.m

```diff
@@ -504,7 +504,30 @@ -(ImageCacheEntry *)entryForKey:(NSURL *)url
 			{
 				scaleUp = YES;
 			}
-			UIImage * resultImage = [UIImage imageWithContentsOfFile:path];
+			UIImage *resultImage = nil;
+			NSRange range = [urlString rangeOfString:@".app"];
+			NSString *imageArg = nil;
+			if (range.location != NSNotFound) {
+				imageArg = [urlString substringFromIndex:range.location+5];
+			}
+			if (imageArg != nil) {
+				unsigned char digest[CC_SHA1_DIGEST_LENGTH];
+				NSData *stringBytes = [imageArg dataUsingEncoding: NSUTF8StringEncoding];
```

Comment:
Yikes you are right. A better way to do this is instead of using ```urlString = [url absoluteString]```, we use ```urlString = [url path]``` instead. since path is the recommended by Apple when dealing with local file paths, and it won't convert spaces to %20. PR coming soon.